styles/sass/main.scss

```diff
@@ -544,592 +571,599 @@ body {
-          padding-bottom: 2rem;
-          background-clip: border-box;
-          -webkit-background-clip: text;
-          color: transparent;
-          background-image: url('https://s3.amazonaws.com/usefulangle/news/95-5f6cc1ad5575e.jpg');
+          p {
+            border: 5px solid white;
+            background-color: $grey;
+            margin-left: 1rem;
+            margin-right: 1rem;
+            padding: 0.8rem;
+          }
+
+          h6 {
+            color: $black;
+            font-size: 20px;
+            font-weight: bold;
+            margin-left: 1rem;
+            margin-right: 1rem;
+            margin-bottom: 1rem;
+            padding-top: 1rem;
+          }
+
+          small {
+            color: $black;
+            font-size: 15px;
+            font-weight: light;
+            margin-left: 1rem;
+          }
         }
+      }
 
-        p {
-          border: 5px solid white;
-          background-color: $grey;
-          margin-left: 1rem;
-          margin-right: 1rem;
-          padding: 0.8rem;
+      .main-program {
+        padding-top: 1.5rem;
+        background-image: url('../../images/pattern_bg.png');
+        background-color: $black;
+        color: white;
+        height: 80vh;
+
+        hr {
+          width: 35px;
+          border: 1px solid $red;
         }
 
-        h6 {
-          color: $black;
-          font-size: 20px;
+        // main-program area
+        .main-program-title {
+          text-align: center;
+          font-size: 25px;
           font-weight: bold;
-          margin-left: 1rem;
-          margin-right: 1rem;
-          margin-bottom: 1rem;
-          padding-top: 1rem;
         }
 
-        h8 {
-          color: $black;
-          font-size: 15px;
-          font-weight: light;
-          margin-left: 1rem;
-        }
-      }
-    }
+        .program-container {
+          display: flex;
+          padding: 30px;
 
-    .main-program {
-      padding-top: 1.5rem;
-      background-image: url('../../images/pattern_bg.png');
-      background-color: $black;
-      color: white;
-      height: 80vh;
+          .program-card {
+            background-color: #3c3f46;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            margin: 5px;
+            padding: 10px;
+            gap: 2px;
+            opacity: 0.7;
+            width: 100%;
+            height: 30vh;
 
-      hr {
-        width: 35px;
-        border: 1px solid $red;
-      }
+            .icon {
+              width: 30%;
+              padding: 5px;
 
-      // main-program area
-      .main-program-title {
-        text-align: center;
-        font-size: 25px;
-        font-weight: bold;
-      }
+              img {
+                width: 50px;
+                height: 50px;
+              }
+            }
 
-      .program-container {
-        display: flex;
-        padding: 30px;
+            .title {
+              width: 35%;
```

Comment:
- [ ] Kindly align your card title by making the width bigger than 35%, it currently looks not too good
![Screenshot from 2022-08-02 22-44-21](https://user-images.githubusercontent.com/66097371/182479375-e384f691-04fc-42c5-9aae-012b161afa7f.png)